libraries/classes/Template.php

```diff
@@ -104,6 +100,10 @@ public static function getTwigEnvironment(string|null $cacheDir): Environment
      */
     private function load(string $templateName): TemplateWrapper
     {
+        if (static::$twig === null) {
+            static::$twig = self::getTwigEnvironment($this->config?->getTempDir('twig'));
```

Comment:
```suggestion
            static::$twig = self::getTwigEnvironment($this->config->getTempDir('twig'));
```
Looks wrong to allow a non existing config

Reply:
Yeah, I know. Moving the getTempDir call out of the constructor is the first step. Now I need to check if it's really possible that config will ever be null.